tests/unit/go/test_app.py

```diff
@@ -94,6 +94,7 @@ def test_go_environment_vars(
         log_files=[],
         metrics_target=f"*:{framework_config.metrics_port}",
         metrics_path=framework_config.metrics_path,
+        unit_name="go/0",
```

Comment:
I saw this change for Flask, Django and go tests, but not for fast API?

Reply:
I think the refactor for the App was done in Go, an that is why that test is there. 

I have reviewed the tests for FastAPI, and added one integration so the env variables for integrations are checked, that was missing.